tests/test_bencode.py

```diff
@@ -99,3 +99,9 @@ def test_encode_complex():
 def test_encode_errors():
     with pytest.raises(BencodeEncodingError):
         Bencode.encode(object())
+
+def test_encode_error_long():
+    # os.path.getsize() can be longs on py2
+    if sys.version_info <= (3, 0):
+        a_long = 741634835L
+        encoded_long = Bencode.encode(a_long)
```

Comment:
No, this simply proves that it works. Without adding long to int_types in the bencode file this would have failed.